### PR TITLE
Release 1.33.1: Improvements on integration with 3rd party services

### PR DESCRIPTION
**Pull request by Gregory-Bessonov**

### CODE REVIEW

**Review by purehchan — COMMENTED** [review]
LGTM but I cannot tell how this fixes the documentation issue. Can you elaborate in the commit message? Thanks

File: pypureclient/flasharray/FA_2_0/models/qos.py [hunk 1]
```diff
@@ -66,8 +66,8 @@ def __setattr__(self, key, value):
             if value < 1048576:
                 raise ValueError("Invalid value for `bandwidth_limit`, must be a value greater than or equal to `1048576`")
         if key == "iops_limit" and value is not None:
-            if value > 104857600:
-                raise ValueError("Invalid value for `iops_limit`, value must be less than or equal to `104857600`")
+            if value > 100000000:
```
[the remark this answers is not in the document]
Comment (Gregory-Bessonov) [reply pypureclient/flasharray/FA_2_0/models/qos.py:69]: It is the fix of specifications only and does not correspond to any specific FA version. I sent you a link to the original ticket in DM.

File: pypureclient/flasharray/FA_2_0/client.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ class Client(object):
     DEFAULT_TIMEOUT = 15.0
     DEFAULT_RETRIES = 5
     # Format: client/client_version/endpoint/endpoint_version/system/release
-    USER_AGENT = USER_AGENT_TEMPLATE.format(prod='FA', sys=platform.system(), rel=platform.release())
+    USER_AGENT = USER_AGENT_TEMPLATE.format(prod='FA', rest_version='2.0', sys=platform.system(), rel=platform.release())
```
Comment (Gregory-Bessonov) [inline pypureclient/flasharray/FA_2_0/client.py:24]: > LGTM but I cannot tell how this fixes the documentation issue. Can you elaborate in the commit message?]

This change fixes the integration. Previous commit introduced `{rest_version}` part in USER_AGENT_TEMPLATE and `rest_version` format option in `client.py` but _only_ in Pure1 part.

It does not affect py-pure-client pypi package, but read-the-doc has it is own way of docs generation. It imports files directly from repo, so having unresolved part in template lead to the import error which in turn broke documentation build.

**Review by purehchan — APPROVED** [review]
LGTM

**Pull request merged**